generated/java/build.gradle

```diff
@@ -1,6 +1,7 @@
 import groovy.io.FileType
 
 def generated_grpc_version = "0.1.5"
+def pubsub_grpc_version = "0.1.6"
```

Comment:
call this `generated_pubsub_grpc_version`.

More generally: I would like to get into the habit of generating the grpc packages all at the same version. It is much easier to manage refreshes if we do that. This change is a case in point: you have to do something special for pubsub because its version is different than the rest.